tests/test_indefinite_freeze_attack.py

```diff
@@ -171,12 +180,26 @@ def tearDown(self):
 
 
   def test_without_tuf(self):
-    # Scenario:
+    # Two tests are conducted here.
```

Comment:
Technically, Test 2 is skipped.  But I see why you'd want to acknowledge the two possible cases for stale metadata.